### PR TITLE
Updated defaults based on IRA Tax Incentives

### PR DESCRIPTION
**Pull request by bpulluta**
### What kind of change does this PR introduce?
(Bug fix, feature, docs update, ...)
REopt defaults are, generally, based off of the latest U.S. tax code. The passage of the Inflation Reduction Act has introduced several changes to available incentives for REopt technologies, including the Investment Tax Credit, MACRS, and Production Tax Credit. This PR introduces changes to align current REopt.jl defaults with these newly available incentives. 

### What is the new behavior (if this is a feature change)?
| Tech                                                                    | Input                    | Default       | New Default          |
|-------------------------------------------------------------------------|--------------------------|---------------|----------------------|
| CHP (Reciprocating engine, Microturbine, Combustine turbine, Fuel cell) | ITC                      | 10%           | 30%                  |
|                                                                         | MACRS schedule           | 5 years       | 5 years (no change)  |
|                                                                         | MACRS bonus depreciation | 100%          | 80%                  |
| Hot water storage                                                       | ITC                      | No ITC input  | 30%                  |
|                                                                         | MACRS schedule           | No MACRS      | 7-year               |
|                                                                         | MACRS bonus depreciation | 0%            | 80%                  |
| Chilled water storage                                                   | ITC                      | No ITC input  | 30%                  |
|                                                                         | MACRS schedule           | No MACRS      | 7-year               |
|                                                                         | MACRS bonus depreciation | 0%            | 80%                  |
| Wind                                                                    | ITC                      | 26%           | 30%                  |
|                                                                         | MACRS schedule           | 5 years       | 5 years (no change)  |
|                                                                         | MACRS bonus depreciation | 100%          | 80%                  |
| PV                                                                      | ITC                      | 26%           | 30%                  |
|                                                                         | MACRS schedule           | 5 years       | 5 years (no change)  |
|                                                                         | MACRS bonus depreciation | 100%          | 80%                  |
| GHP                                                                     | ITC                      | 10%           | 30%                  |
|                                                                         | MACRS schedule           | 5 years       | 5 years (no change)  |
|                                                                         | MACRS bonus depreciation | 100%          | 80%                  |
| Battery                                                                 | ITC                      | 0%            | 30%                  |
|                                                                         | MACRS schedule           | 7 years       | 7 years (no change)  |
|                                                                         | MACRS bonus depreciation | 100%          | 80%                  |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # REopt Julia package
-This package is currently under development, but it now has **most** of capabilities of the REopt model used in the [REopt API](https://github.com/NREL/REopt_API). We expect to have the first stable release by Fall 2022.
+This package is currently under development, but it now has all of the capabilities of the REopt model used in the [REopt API](https://github.com/NREL/REopt_API). We expect to have the first stable release by Summer 2023.
```
Comment (adfarth) [inline README.md:2]: @bpulluta are we sure it currently has all? I believe we're still missing a few techs, although they should be in there by the Summer 2023 release

Comment (bpulluta) [reply README.md:2]: @adfarth See b37fcde8f592c3736e58e5c838b82fc021919. It seems that this was an update from 2-weeks ago. Should I revert to the previous description?

Comment (bpulluta) [reply README.md]: @adfarth not sure how this ended up in my branch but I reverted back to the old timeline

File: src/core/energy_storage/thermal_storage.jl [hunk 2]
```diff
@@ -63,10 +63,10 @@ Base.@kwdef struct ColdThermalStorageDefaults <: AbstractThermalStorageDefaults
     installed_cost_per_gal::Float64 = 1.50
     thermal_decay_rate_fraction::Float64 = 0.0004
     om_cost_per_gal::Float64 = 0.0
-    macrs_option_years::Int = 0
-    macrs_bonus_fraction::Float64 = 0.0
+    macrs_option_years::Int = 7
+    macrs_bonus_fraction::Float64 = 0.8
     macrs_itc_reduction::Float64 = 0.0
```
[the remark this answers is not in the document]
Comment (bpulluta) [reply src/core/energy_storage/thermal_storage.jl]: @adfarth addressed in 4a93e294125ed87b18768fe3860a8db07a43a349 & 30f4b20efcb9d5f916dc27b3e2907b1d2feb8d76

File: CHANGELOG.md [hunk 3]
```diff
@@ -22,7 +22,14 @@ Classify the change according to the following categories:
...
     ### Removed
-
+    
+## Develop - 2023-01-31
+ ##### Changed
+ - Updated the following default values:
+   - PV, Wind, Storage, CHP, GHP, Hot Water Storage, Cold Water Storage, Electric Storage: **federal_itc_fraction(PV,Wind, CHP,GHP)** and **total_itc_fraction(Hot Water Storage, Cold Water Storage, Electric Storage)** to 0.3 (30%)
+   - PV, Wind, Storage, CHP, GHP, Hot Water Storage, Cold Water Storage, Electric Storage: ***macrs_bonus_fraction** to 0.8 (80%)
+   - Wind ITC no longer determined based on size class
+   - Hot Water Storage and Cold Water Storage: **macrs_itc_reduction** to 0.5 (50%)
```
Comment (adfarth) [inline CHANGELOG.md:32]: @bpulluta could you add the update the Hot and Cold Water Storage macrs_option_years here as well?

**Review by adfarth — APPROVED** [review]
@bpulluta All looks good to me! I committed just one small change, so once the tests from that pass you should be good to merge :)

File: CHANGELOG.md (hunk 3 above)
Comment (bpulluta) [reply CHANGELOG.md]: @adfarth addressed in 017201c953531f6ec824d5b726ea72a69dc11b94

**Pull request merged**